android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1431,8 +1431,9 @@ public void onPlaybackParametersChanged(PlaybackParameters params) {
 
     @Override
     public void onVolumeChanged(float volume) {
-        eventEmitter.volumeChange(volume);
-    }
+//        todo
```

Comment:
tado here

Reply:
I think onVolumeChanged and onAdError are new APIs
So I have to implement it for Fabric.
I will let you know when it's implemented.